[debug] [2025-03-05T19:22:45.506Z] ----------------------------------------------------------------------
[debug] [2025-03-05T19:22:45.508Z] Command:       /opt/homebrew/Cellar/node/23.9.0/bin/node /opt/homebrew/bin/firebase login
[debug] [2025-03-05T19:22:45.508Z] CLI Version:   13.32.0
[debug] [2025-03-05T19:22:45.508Z] Platform:      darwin
[debug] [2025-03-05T19:22:45.508Z] Node Version:  v23.9.0
[debug] [2025-03-05T19:22:45.509Z] Time:          Wed Mar 05 2025 11:22:45 GMT-0800 (Pacific Standard Time)
[debug] [2025-03-05T19:22:45.509Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-03-05T19:22:45.510Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2025-03-05T19:22:45.797Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2025-03-05T19:22:45.798Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-03-05T19:22:45.798Z] Failed to fetch MOTD Error: EACCES: permission denied, mkdir '/Users/mykyta/.config/configstore'
You don't have access to this file.

[debug] [2025-03-05T19:23:00.739Z] Error: EACCES: permission denied, mkdir '/Users/mykyta/.config/configstore'
You don't have access to this file.

    at Object.mkdirSync (node:fs:1361:26)
    at module.exports.sync (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/make-dir/index.js:114:6)
    at set all (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:59:12)
    at Configstore.set (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:91:12)
    at Command.actionFn (/opt/homebrew/lib/node_modules/firebase-tools/lib/commands/login.js:36:35)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2025-03-05T19:23:01.000Z] Error: EACCES: permission denied, mkdir '/Users/mykyta/.config/configstore'
You don't have access to this file.

    at Object.mkdirSync (node:fs:1361:26)
    at module.exports.sync (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/make-dir/index.js:114:6)
    at set all (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:59:12)
    at Configstore.set (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:91:12)
    at process.<anonymous> (/opt/homebrew/lib/node_modules/firebase-tools/lib/bin/firebase.js:100:35)
    at process.emit (node:events:519:35)
    at process.emit (node:domain:489:12)
    at process.exit (node:internal/process/per_thread:226:15)
    at Timeout._onTimeout (/opt/homebrew/lib/node_modules/firebase-tools/lib/errorOut.js:20:17)
    at listOnTimeout (node:internal/timers:614:17)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2025-03-05T19:23:04.896Z] ----------------------------------------------------------------------
[debug] [2025-03-05T19:23:04.897Z] Command:       /opt/homebrew/Cellar/node/23.9.0/bin/node /opt/homebrew/bin/firebase login
[debug] [2025-03-05T19:23:04.898Z] CLI Version:   13.32.0
[debug] [2025-03-05T19:23:04.898Z] Platform:      darwin
[debug] [2025-03-05T19:23:04.898Z] Node Version:  v23.9.0
[debug] [2025-03-05T19:23:04.898Z] Time:          Wed Mar 05 2025 11:23:04 GMT-0800 (Pacific Standard Time)
[debug] [2025-03-05T19:23:04.898Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-03-05T19:23:04.899Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2025-03-05T19:23:05.052Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2025-03-05T19:23:05.052Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-03-05T19:23:05.053Z] Failed to fetch MOTD Error: EACCES: permission denied, mkdir '/Users/mykyta/.config/configstore'
You don't have access to this file.

[debug] [2025-03-05T19:23:07.125Z] Error: EACCES: permission denied, mkdir '/Users/mykyta/.config/configstore'
You don't have access to this file.

    at Object.mkdirSync (node:fs:1361:26)
    at module.exports.sync (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/make-dir/index.js:114:6)
    at set all (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:59:12)
    at Configstore.set (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:91:12)
    at Command.actionFn (/opt/homebrew/lib/node_modules/firebase-tools/lib/commands/login.js:36:35)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2025-03-05T19:23:07.383Z] Error: EACCES: permission denied, mkdir '/Users/mykyta/.config/configstore'
You don't have access to this file.

    at Object.mkdirSync (node:fs:1361:26)
    at module.exports.sync (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/make-dir/index.js:114:6)
    at set all (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:59:12)
    at Configstore.set (/opt/homebrew/lib/node_modules/firebase-tools/node_modules/configstore/index.js:91:12)
    at process.<anonymous> (/opt/homebrew/lib/node_modules/firebase-tools/lib/bin/firebase.js:100:35)
    at process.emit (node:events:519:35)
    at process.emit (node:domain:489:12)
    at process.exit (node:internal/process/per_thread:226:15)
    at Timeout._onTimeout (/opt/homebrew/lib/node_modules/firebase-tools/lib/errorOut.js:20:17)
    at listOnTimeout (node:internal/timers:614:17)
[error] 
[error] Error: An unexpected error has occurred.
